jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/TextFieldsTests.java

```diff
@@ -139,69 +121,30 @@ public void hideDetailsTextFieldTest() {
 
     @Test
     public void hintTextFieldTest() {
-        hintTextField.forEach(textField -> {
-            textField.is().noMessage();
-            textField.focus();
-            textField.is().message();
-        });
+        String hintText = "www.example.com/page";
 
-        visibleHintTextField.forEach(textField -> {
-            textField.message().has().text("www.example.com/page");
-            textField.focus();
-            textField.message().has().text("www.example.com/page");
-        });
-    }
-
-    @Test
-    public void iconsTextFieldTest() {
-        String mdiMapMarker = "mdi-map-marker";
-        for (int index = 0; index <= 12; index += 4) {
-            iconsTextField.get(1 + index).getPrependOuterIcon().is().type(mdiMapMarker);
-            iconsTextField.get(2 + index).getPrependInnerIcon().is().type(mdiMapMarker);
-            iconsTextField.get(3 + index).getAppendInnerIcon().is().type(mdiMapMarker);
-            iconsTextField.get(4 + index).getAppendOuterIcon().is().type(mdiMapMarker);
-        }
-    }
+        hintTextField.get(1).is().noMessage();
+        hintTextField.get(1).focus();
+        hintTextField.get(1).is().message();
 
-    @Test
-    public void outlinedTextFieldTest() {
-        outlinedTextField.get(1).is().outlined();
-        outlinedTextField.get(2).is().outlined();
-    }
+        hintTextField.get(2).is().noMessage();
+        hintTextField.get(2).focus();
+        hintTextField.get(2).is().message();
 
-    @Test
-    public void prefixesAndSuffixesTextFieldTest() {
-        prefixesAndSuffixesTextField.get(1).prefix().has().text("$");
-        prefixesAndSuffixesTextField.get(2).suffix().has().text("lbs");
-        prefixesAndSuffixesTextField.get(3).suffix().has().text("@gmail.com");
-        prefixesAndSuffixesTextField.get(4).suffix().has().text("PST");
-    }
+        visibleHintTextField.get(1).message().has().text(hintText);
+        visibleHintTextField.get(1).focus();
+        visibleHintTextField.get(1).message().has().text(hintText);
 
-    @Test
-    public void shapedTextFieldTest() {
-        shapedTextField.get(1).is().shaped();
-        shapedTextField.get(2).is().shaped();
-        shapedTextField.get(1).is().css("border-radius", "16px 16px 0px 0px");
-        shapedTextField.get(2).is().css("border-radius", "16px 16px 0px 0px");
+        visibleHintTextField.get(2).message().has().text(hintText);
+        visibleHintTextField.get(2).focus();
+        visibleHintTextField.get(2).message().has().text(hintText);
```

Comment:
If you are working with List elements, use a local variable for the list/element you are working with